execution-engine/src/engines/wasmi.rs

```diff
@@ -86,11 +86,7 @@ impl MemoryHandler for MemoryRef {
     type Slice = &'static [u8];
     type SliceMut = &'static mut [u8];
 
-    fn get_slice<'a>(
-        &'a self,
-        address: u32,
-        length: u32,
-    ) -> FileSystemResult<Bound<'a, Self::Slice>> {
+    fn get_slice(&self, address: u32, length: u32) -> FileSystemResult<Bound<Self::Slice>> {
```

Comment:
Which format for this function prototype is more "rusty"? I definitely prefer the new one, but do the rust tools have an opinion on this?

Reply:
It is the `cargo fmt`. I believe they make it one line since the parameter list is not too long.